show/main.py

```diff
@@ -1764,13 +1764,31 @@ def uptime(verbose):
     cmd = "uptime -p"
     run_command(cmd, display_cmd=verbose)
 
-@cli.command()
-@click.option('--verbose', is_flag=True, help="Enable verbose output")
-def clock(verbose):
+
+#
+# 'clock' command group ("show clock ...")
+#
+@cli.group('clock', invoke_without_command=True)
+@click.pass_context
+def clock(ctx):
     """Show date and time"""
-    cmd ="date"
-    run_command(cmd, display_cmd=verbose)
+    # If invoking subcomand, no need to do anything
+    if ctx.invoked_subcommand is not None:
+        return
+
+    run_command('date')
+
+
+@clock.command()
+def timezones():
+    """List of available timezones"""
+    run_command('timedatectl list-timezones')
+    click.echo('Etc/UTC')
```

Comment:
Why do we add Etc/UTC at the end? The timedatectl command list-timezones lists UTC

Reply:
Yes, it does, but `Etc/UTC` is also a valid timezone which is not listed there.
That is why I add it there